templates/stats/publications.tpl

```diff
@@ -233,13 +233,16 @@
 								</action-panel>
 								{if $geoReportType}
 									<action-panel class="pkpStats__reportAction">
-										<h2 id="report-type-article">{translate key="common.geographic"}</h2>
-										<p id="report-type-article-description">
+										<h2 id="report-type-geographic">{translate key="common.geographic"}</h2>
+										<p id="report-type-geographic-description">
 											{translate key="stats.publications.downloadReport.downloadGeographic.description"}
 										</p>
+										<p id="report-type-geographic-ccAttribution">
+											{translate key="stats.geographic.ccAttribution"}
+										</p>
 										<template slot="actions">
 											<pkp-button
-												aria-describedby="report-type-article report-type-article-description"
+												aria-describedby="report-type-geographic report-type-geographic-description"
```

Comment:
I think my example code probably included the `aria-describedby` attributes but I don't think we need them (or the `id` attributes they reference). That's because the download buttons on the publications and issue stats pages all have descriptive labels: Download Articles, Download Files, etc.

We would use the `aria-describedby` labels in cases where we had several buttons with identical names -- like Download, Download, Download -- and needed to distinguish them for screen readers.